pkg/site/site.go

```diff
@@ -475,6 +480,11 @@ func ReadSiteConfig(siteConfig *corev1.ConfigMap, namespace string, defaultIngre
 	var errs []string
 	var result types.SiteConfig
 	result.Spec.SkupperNamespace = siteConfig.Namespace
+
+	annotationSeparator := ","
+	if _, ok := siteConfig.Data[SiteConfigAnnotationSeparator]; ok {
```

Comment:
Minor nitpick: looks odd to ignore the value and then re-retrieve it

Reply:
fixed